src/components/FeedbackItem.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import React from "react";
+import { Box, Typography } from "@mui/material";
+
+interface FeedbackItemProps {
+	name: string;
+	timeAgo: string;
+	feedback: string;
+}
+
+const FeedbackItem: React.FC<FeedbackItemProps> = ({
+	name,
```

Comment:
Breaking each feedback item into its own FeedbackItem component makes the code so much cleaner, this is a great idea. I’d suggest using a unique identifier as the key instead of index if possible, this could help with rendering performance, but might be a bit of additional work.

Reply:
That is a really great point!